### DISCUSSION TITLE
Roll don't have root motion after retarget to Metahuman

### DISCUSSION THREAD

**Discussion by kevinbear91** [Sixze/ALS-Refactored#123] [Q&A]
Hello,

After retarget the Animtions and fix the refences for ABP, and now only a problem is roll the character don't have root motions. the CS_Defualt file I duplicated to my Metahuman, and fixed montage refences also. I suspect Als Set Root Motion Scale notify state not working. But  I don't know where can check it. 

My Metahuman .h&.cpp file copy the content from AlsCharacterExample.h & .cpp, is still have something else I missed?

![Snipaste_2022-07-23_17-25-30](https://user-images.githubusercontent.com/7070983/180599376-7e7d203e-66ec-4982-a823-075ecca2466d.jpg)

![Snipaste_2022-07-23_17-26-32](https://user-images.githubusercontent.com/7070983/180599414-f1c9ecb3-3fa1-428b-b4e2-7551abf495ed.jpg)

![ezgif-5-06391a284c](https://user-images.githubusercontent.com/7070983/180599200-16a1ccf5-daf8-4a9a-a28e-b3b484b41d5c.gif)

**Comment by PaulNguyen88** [comment]
Same issue!

**Reply by Amegaku** [reply]
it was not the correct retargeting of animations.
fix "https://www.youtube.com/watch?v=cVKgCgcRO5k&t=1s"
